### ISSUE TITLE
Errors on Tensorflow 1.0

### ISSUE THREAD

**Issue by yinhuagang** [llSourcell/rnn_tutorial#2]
However can be solved through tf's tool: https://github.com/tensorflow/tensorflow/tree/r1.0/tensorflow/tools/compatibility

**Comment by alinabee** [comment]
You can get it to work by modifying the following lines:

Replace tf.unpack() with tf.unstack():
```
inputs_series = tf.unstack(batchX_placeholder, axis=1)
labels_series = tf.unstack(batchY_placeholder, axis=1)
```

Swap the order of arguments as below:
`input_and_state_concatenated = tf.concat([current_input, current_state], 1)`

Add argument labels:
`losses = [tf.nn.sparse_softmax_cross_entropy_with_logits(logits=logits, labels=labels) for logits, labels in zip(logits_series, labels_series)]`

Eliminate the warning with this replacement:
`sess.run(tf.global_variables_initializer()) #instead of sess.run(tf.initialize_all_variables())`
